examples/todoapp/test/feature_statistics/view/statistics_golden_test.dart

```diff
@@ -0,0 +1,19 @@
+import '../../Stubs.dart';
+import '../../helpers/golden_helper.dart';
+import '../../helpers/models/scenario.dart';
+import '../factory/statistics_factory.dart';
+
+void main() {
+  runGoldenTests([
+    generateDeviceBuilder(
+        widget: statisticsFactory(todosStats: Stubs.todoListStatisticsEmpty),
+        //example: Stubs.emptyList
+        scenario: Scenario(name: 'statistics_empty')),
+    generateDeviceBuilder(
+        widget: statisticsFactory(todosStats: Stubs.todoListStatistics), //example:  Stubs.success
+        scenario: Scenario(name: 'statistics_success')),
+    generateDeviceBuilder(
+        widget: statisticsFactory(), //loading
```

Comment:
I guess the comments were left over from testing or documenting things. Please make sure you remove them.